### ISSUE TITLE
net::ERR_CERT_COMMON_NAME_INVALID

### ISSUE THREAD

**Issue by lauyeung** [orthes/medium-editor-insert-plugin#496]
I was trying to add a youtube link to the demo here but it looks like the certificate is invalid: http://linkesch.com/medium-editor-insert-plugin/

![image](https://user-images.githubusercontent.com/3825959/66430899-da363b00-e9cf-11e9-8a98-bd7c505df820.png)

**Comment by boussou** [comment]
no, I think it is because this plugin uses https://iframely.com and now this service is no longer free.

**Comment by boussou** [comment]
I added a bug report linked to yours here: https://github.com/orthes/medium-editor-insert-plugin/issues/497

For your own problem described here, you would use the http instead of https address - you can  set this using  oembedProxy

eg

 oembedProxy: 'http://medium.iframe.ly/api/oembed?iframe=1',
